submit.tsa.slurm

```diff
@@ -23,11 +23,8 @@ export MALLOC_TRIM_THRESHOLD_=536870912
 unset G2G
 export MV2_USE_GPUDIRECT=0 #with this perf are ok
 
-if [ ! -z $COSMO_TESTENV ] ; then
-  # UCX segfaults
-  export UCX_MEMTYPE_CACHE=n
-fi
-
+# UCX segfaults
+export UCX_MEMTYPE_CACHE=n
```

Comment:
I think this is not necessary anymore, since this is exported in the modules file in env.tsa.sh